### ISSUE TITLE
Cocoapods Podspec missing source_files / preserve_paths

### ISSUE THREAD

**Issue by andygeers** [couchbase/couchbase-lite-ios#1205]
I'm working on a Cocoapods Podspec for a framework that expresses a dependency on couchbase-lite-ios. I find that I can't use the official couchbase-lite-ios.podspec because it doesn't properly define where the header files live, and I end up with lots of errors along the lines of "Unknown path <CouchbaseLite/CouchbaseLite.h>. If I create my own forked podspec for Couchbase Lite then I can get it working just fine, by adding the following lines:

```
  "source_files": "couchbase-lite-ios/CouchbaseLite.framework/Headers/*.h",
  "preserve_paths": ["LICENSE.txt", "couchbase-lite-ios/CouchbaseLite.framework"],
```

Would it be possible to get these into the official Podspec?
--------
* Version:   1.2.0
* Client OS: iOS 9.1.3
* Server:    N/A

**Comment by pasin** [comment]
@andygeers I have one question for the `preserve_paths`. Currently we have the preserve_paths configured without `couchbase-lite-ios/`like [this](https://github.com/couchbase/couchbase-lite-ios/blob/master/Podspec/couchbase-lite-ios.podspec#L18). 

Does it require to have the `couchbase-lite-ios/`folder prefix?

**Comment by pasin** [comment]
Also in the binary zip file, `CouchbaseLite.framework` is at the root so I'm not sure if `couchbase-lite-ios/` is required.

**Comment by pasin** [comment]
@andygeers I have modified the couchbase-lite-ios podspec for the _unofficial_ 1.2.1 release by including the `source_files` pointing to the .h files. Would it be possible to try this out and see if it works for you (see the gist below)? Please let me know.

https://gist.github.com/pasin/fbd27dd3caee4dcabc20dddfc6d380db

**Comment by pasin** [comment]
I have just published cocoapod v1.2.1. I included `source_files` for couchbase-lite-ios. I backed out the change for couchbase-lite-osx as it couldn't pass the lint. I'm closing this ticket now so please report back  if the issue still persists.

**Comment by snej** [comment]
Oops, didn't mean to reopen this!
